app/assets/javascripts/views/restrooms/restrooms.js.coffee

```diff
@@ -3,15 +3,20 @@
 # You can use CoffeeScript in this file: http://coffeescript.org/
 
 $ ->
-  $('#ada_filter').click -> 
+  $('#ada_filter').click ->
     if $(this).hasClass("active")
       $('.listItem.not_accessible').show()
-    else 
+    else
       $('.listItem.not_accessible').hide()
-    
-  $('#unisex_filter').click -> 
+
+  $('#unisex_filter').click ->
     if $(this).hasClass("active")
       $('.listItem.not_unisex').show()
-    else 
+    else
       $('.listItem.not_unisex').hide()
 
+  $('#changing_table_filter').click ->
```

Comment:
It's possible that we'll want to start pulling out all these boolean icons to enums rather than listing all of them here. Doesn't need to be part of this though, just a follow up. By no means is three icons unwieldy :P